app/views/shelters/show.html.erb

```diff
@@ -17,4 +17,18 @@
       Zip: <%= @shelter.zip %><br/>
     </p>
   </section>
+
+
+  <% @shelter.reviews.each do |review| %>
+  <section id = "review-<%= review.id %>">
+    <p>
+      Title: <%= review.title %> <br/>
+      Rating: <%= review.rating%> <br/>
+      Content: <%= review.content %> <br/>
+      <%= image_tag "#{review.image_path}", alt: "#{review.image_path}" %>
+    </p>
+  </section>
+  <% end %>
+
+  <a href="/shelters/<%= @shelter.id %>/reviews/new">Create New Review</a>
```

Comment:
Can you move this up to the navbar section in the #page-header for consistency? Thanks!

Reply:
Completed.